tests/component-tests/textfieldSlider/js/TextfieldSliderTests.js

```diff
@@ -39,93 +39,67 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             jqUnit.assertEquals("The ARIA min should be 0", 0, thumb.attr("aria-valuemin"));
         });
 
-        var testSetting = function (valToTest, expected) {
+        var testInputField = function (valToTest, expected) {
             var slider = $(".flc-textfieldSlider-slider");
             var textfield = $(".flc-textfieldSlider-field");
             var thumb = $(".ui-slider-handle");
 
-            slider.slider("value", valToTest);
-            jqUnit.assertEquals("Slider value should be " + expected, expected, slider.slider("value"));
             textfield.val(valToTest);
             textfield.change();
             jqUnit.assertEquals("Textfield value should be the " + expected, expected, textfield.val());
             jqUnit.assertEquals("The ARIA value now should be " + expected, expected, thumb.attr("aria-valuenow"));
+            jqUnit.assertEquals("Slider value should be " + expected, expected, slider.slider("value"));
+        };
+
+        var testSlider = function (valToTest, expected) {
```

Comment:
Sorry not to catch this in earlier rounds of the review - could you put all these global functions into a fluid.tests.textfieldSlider namespace? Thanks